student.rb

```diff
@@ -15,7 +15,7 @@ class Student < Person
   # classroom - The String name of the classroom.
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
-    @classroom = classroom
+    @classroom = Classroom.new(classroom)
   end
 
```

Comment:
- [ ] Kindly ensure to create a belong-to association to indicate that a student belongs to the classroom, similar to the code below:

```ruby
class Student < Person
  attr_reader :classroom
  .
  .

  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
end
```